system/security/shared/caching.cpp

```diff
@@ -624,7 +626,7 @@ bool CPermissionsCache::queryPermsManagedFileScope(ISecUser& sec_user, const cha
                 {
                     *accessFlags = res->getAccessFlags();
                     managedScope.append(const_cast<char *>(res->getName()));
-                    DBGLOG("FileScope %s for %s(%s) access denied %d at scope %s, took %dms",fullScope, sec_user.getName(), res->getName(), *accessFlags, scope, msTick()-start);
+                    PROGLOG("FileScope %s for %s(%s) access denied %s(%d) at scope %s, took %dms", fullScope, sec_user.getName(), res->getName(), getSecAccessFlagName(*accessFlags), *accessFlags, scope, msTick()-start);
```

Comment:
MCoperatorProgress/MCoperatorInfo seems more appropriate (or MCauditInfo).

Reply:
Updated to MCoperatorProgress. My main concern is this log message will be extremely useful for the operations team to research file scope denials themselves instead of having the platform team do it.